javaee/community/configmap/booster.yaml

```diff
@@ -0,0 +1,16 @@
+metadata:
+name: JavaEE - Config Map
+description: Simple HTTP endpoint where the WildFly application uses OpenShift ConfigMap to retrieve application parameters.
+source:
+  git:
+    url: https://github.com/wildfly-booster-for-openshift/wildfly-configmap
+    ref: master
+environment:
+  staging:
+    source:
+      git:
+        ref: master
+  production:
+    source:
+      git:
+        ref: master
```

Comment:
Since the booster-catalog is tagged when pushing to production, it is advisable to tag the booster instead of using `master`